assets/styles/index.css

```diff
@@ -21,8 +16,12 @@ body {
     src: url(Rubik-Bold.ttf);
 }
 
+body {
+    /* apply default styles for body */
```

Comment:
please delete unnecessary comment

Reply:
it's not unnecessary, that comment is to signify that the styles that applied to body is the default styles that come from `tailwind.configuration.js` file and can be overwritten because the specificity is not important, using `@apply` keyword to apply classes from configuration file

Reply:
Okay, LGTM
Please merge it to staging then